src/mongoc/mongoc-topology.c

```diff
@@ -399,7 +433,7 @@ mongoc_topology_server_by_id (mongoc_topology_t *topology, uint32_t id)
  */
 
 void
-_mongoc_topology_request_scan (mongoc_topology_t *topology)
+mongoc_topology_request_scan (mongoc_topology_t *topology)
```

Comment:
in retrospect, we should be able to call this from topology_select, but it's just not right (it shouldn't start the scan and shouldn't possibly block).